release-team/role-handbooks/branch-manager/README.md

```diff
@@ -115,6 +116,42 @@ As the first build from the new release branch, publication this build does not
 Adding the --rc flag switches behavior on to building release candidates.  Again the gcbmgr/anago scripting automatically finds and increments the current build number.
 * `./gcbmgr stage release-1.12 --rc --build-at-head --nomock`
 
+In an perfect world, `rc.1` and the official release are the same commit. To get as close to that as we can the following thing should be considered:
```

Comment:
Perhaps instead of "To get as close to that as we can ..."  say:

"To get as close to the official release as possible **,** the following should be considered:  " ?